MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,284 @@
+        
+        overviewText.textColor = .black
+        overviewText.text = details?.summary
+        overviewText.autoSetDimensions(to: CGSize(width: 358, height: 64))
+        overviewText.font = UIFont.systemFont(ofSize: 14)
+        
+        overviewText.autoPinEdge(toSuperviewEdge: .leading, withInset: 16)
+        overviewText.autoPinEdge(.top, to: .bottom, of: overviewLabel, withOffset: 8.38)
+        overviewText.isEditable = false
+        
+    }
+    func buildView(){
+        view.backgroundColor = .white
+        imageFunc()
+        rating()
+        userScore()
+        name()
+        year()
+        genre()
+        favourite()
+        overview()
+        overviewScroll()
+    }
+    
+    func formatTime(minutes: Int) -> String {
+        let sati = minutes / 60
+        let min = minutes % 60
+        return String(format: "%2dh %2dm", sati,min)
+    }
+    
+    func collection () {
+        let flowlayout = UICollectionViewFlowLayout()
+        flowlayout.scrollDirection = .vertical
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: flowlayout )
+        view.addSubview(collectionView)
+        
+        let spacing:CGFloat = 16
+        flowlayout.minimumInteritemSpacing = CGFloat(spacing)
+        flowlayout.minimumLineSpacing = CGFloat(spacing)
+        
+        collectionView.autoPinEdge(toSuperviewEdge: .top, withInset: 480)
+        collectionView.autoPinEdge(toSuperviewEdge: .leading)
+        collectionView.autoPinEdge(toSuperviewEdge: .trailing)
+        
+        collectionView.autoSetDimension(.height, toSize: 104)
+        collectionView.register(MyCell.self, forCellWithReuseIdentifier: "cell")
+        collectionView.dataSource = self
+        collectionView.delegate = self
+    }
+}
+
+extension MovieDetailsViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return details?.crewMembers.count ?? 0
+    }
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        1
+    }
+    internal func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath) as! MyCell
+        let data = details?.crewMembers[indexPath.item]
+        cell.titleLabel.text = data?.name
+        cell.subtitleLabel.text = data?.role
+        return cell
+    }
+}
+extension MovieDetailsViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let spacing: CGFloat = 16
+        let width = (collectionView.bounds.width - 4 * spacing) / 3
+        return CGSize(width: width, height: 40)
+    }
+}
+
+extension UIImageView {
```

Comment:
Extensions should be put in a dedicated folder. With a naming convention for extension files, ie `UIImageView+Extensions`.